language/llama2-70b/main.py

```diff
@@ -146,8 +152,8 @@ def main():
     settings = lg.TestSettings()
     settings.scenario = scenario_map[args.scenario.lower()]
     # mlperf.conf is automatically loaded by the loadgen
-    # settings.FromConfig(args.mlperf_conf, "llama2-70b", args.scenario)
-    settings.FromConfig(args.user_conf, "llama2-70b", args.scenario)
+    # settings.FromConfig(args.mlperf_conf, args.lg_model_name, args.scenario)
+    settings.FromConfig(args.user_conf, args.lg_model_name, args.scenario)
```

Comment:
Should we still comment out the `llama2-70b-interactive` if we are loading from the user config here?

Reply:
Actually, do we still need the user.conf if the mlperf.conf already has the new latency?

Reply:
Mmmm there might be other settings that submitters could modify. So, leaving the user.conf there might be helpful

Reply:
You are right, nvm